spark/src/test/java/org/apache/iceberg/spark/source/TestORCWrite.java

```diff
@@ -0,0 +1,99 @@
+package org.apache.iceberg.spark.source;
+
+import com.google.common.collect.ImmutableMap;
```

Comment:
@shawnding  what are your thoughts on making the class `org.apache.iceberg.spark.source.TestParquetWrite` parameterized and making it work for both Parquet and ORC. Seems like that's less work and it also tests with a lot more test cases. Thoughts? 

If you want to see some examples of parameterized tests, please see `org.apache.iceberg.data.TestLocalScan`

Reply:
Ok thanks Rdsr, that I will rewrite it.